e2e-tests/playwright/utils/UIhelper.ts

```diff
@@ -316,4 +317,17 @@ export class UIhelper {
       expect(color).toBe(expectedRgbColor);
     }
   }
+
+  async goToBackstageJanusProjectCITab() {
```

Comment:
Can we relocate this method to the appropriate page within e2e-tests/playwright/support/pages? It handles navigations and other actions that are more suited for the page it belongs to.